apps/comps/package.json

```diff
@@ -7,8 +7,8 @@
     "dev": "next dev --turbopack --port 3001",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint --max-warnings 0",
     "check-types": "tsc --noEmit",
+    "lint": "eslint --max-warnings 0 .",
```

Comment:
what lint setting do you have in your IDE that alphabetically reorders the `package.json` imports and scripts? if possible, we should add it to our monorepo's config